scripts/scanDependents/README.md

```diff
@@ -32,32 +32,26 @@ Pass it to the command like this
 yarn scan-dependents --token YOUR_PERSONAL_ACCESS_TOKEN
 ```
 
-## Scan versions of the design systems in use
+## Common formulas
 
-This is the default behavior of the script if no additional options are specified, the script will print out design system versions in use by our dependents.
-
-## Scan dependencies of dependents
-
-You can scan for which versions of certain dependencies our dependents are using with
+### Scan for the version of React our dependencies are using
 
 ```
-yarn scan-dependents --dependency <dependency-package-name>
+yarn scan-dependents --dependency react
 ```
 
-For example, this would scan for which version of React our dependencies are using:
+### Scan only a subset of our design systems
 
 ```
-yarn scan-dependents --dependency react
+yarn scan-dependents --designSystems @cmsgov/ds-medicare-gov @cmsgov/ds-healthcare-gov --dependency node-sass
 ```
 
-## Additional options
-
-### Scanning only a subset of our design systems
+This will scan only dependents of the medicare and healthcare design systems to find out what version of `node-sass` they're using if any.
 
-Example:
+### Search JavaScript files for certain strings
 
 ```
-yarn scan-dependents --designSystems @cmsgov/design-system @cmsgov/ds-healthcare-gov --dependency node-sass
+yarn scan-dependents search ChoiceList --extensions js jsx tsx
```

Comment:
It doesn't look like extensions was a part of this change, I don't see it in the available options in index, though it is presented in the search function as an optional prop, something to add in the future?

Reply:
Is it listed when you run it with the help flag? Should be coming from https://github.com/CMSgov/design-system/pull/1971/files#diff-2aca4f9afa500c949a5d976373ed3303a8c6b2ec6bc12ad271b11239f7c35012R72

Reply:
Hmm no it doesn't show up with `--help` for me, though running `yarn scan-dependents search ChoiceList --extensions js jsx tsx` did work for me this time around.